working/templates/srmfunction/SrmFunction.sln

```diff
@@ -34,6 +34,27 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Actions", "Actions", "{8A00
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "$SCRIPTNAME$_1", "$SCRIPTNAME$_1\$SCRIPTNAME$_1.csproj", "{C99E3ABD-DE32-45FA-9FB7-8C9B659E11C1}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Functions", "Functions", "{0A65DF85-1067-44AA-B14B-451BEF792C9D}"
+	ProjectSection(SolutionItems) = preProject
+		Functions\Functions.xml = Functions\Functions.xml
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Definitions", "Definitions", "{32075BA8-6E1E-476A-AA0F-5EB0933E15A2}"
+	ProjectSection(SolutionItems) = preProject
+		Profile Data\Definitions\$PROFILEDEFINITIONNAME$.json = Profile Data\Definitions\$PROFILEDEFINITIONNAME$.json
+		Profile Data\Definitions\readme.md = Profile Data\Definitions\readme.md
+	EndProjectSection
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Instances", "Instances", "{37441868-41FC-42BB-9C33-3B53C1E47D3A}"
```

Comment:
GUID 37441868-41FC-42BB-9C33-3B53C1E47D3A should be added to the guids list in the template.json file